secretstores/azure/keyvault/keyvault.go

```diff
@@ -40,20 +40,8 @@ func NewAzureKeyvaultSecretStore() secretstores.SecretStore {
 
 // Init creates a Kubernetes client
 func (k *keyvaultSecretStore) Init(metadata secretstores.Metadata) error {
-	props := metadata.Properties
-	k.vaultName = props[componentVaultName]
-	certFilePath := props[componentSPNCertificateFile]
-	certBytes := []byte(props[componentSPNCertificate])
-	certPassword := props[componentSPNCertificatePassword]
-
-	k.clientAuthorizer = NewClientAuthorizer(
-		certFilePath,
-		certBytes,
-		certPassword,
-		props[componentSPNClientID],
-		props[componentSPNTenantID])
-
-	authorizer, err := k.clientAuthorizer.Authorizer()
+	k.metadata = metadata
+	authorizer, err := k.GetAuthorizer()
```

Comment:
This will be changed to then :
authorizer, err := k.GetAuthorizer(settings)

Reply:
Solved in commit 8225aae.

Thanks for the review, this is a cleaner solution.